tests/WorldStorage/TreeNBTTest.cpp

```diff
@@ -0,0 +1,228 @@
+
+#include "Globals.h"
+#include "WorldStorage/TreeNBT.h"
+
```

Comment:
```suggestion

#include "../TestHelpers.h"
```

The new test "engine" wasn't implemented there
The rest of the file would also require some attention